### PR TITLE
CSV Import: update default category

### PR DESCRIPTION
**Pull request by Zengraph**
adding more code for forcing default cat to update in product CSV import/update
reference to this issue :  #1058 

forum : https://forum.thirtybees.com/topic/3478-solved-issue-request-created-please-help-csv-product-import-problem-moving-category-issue-product-breadcrumb-not-updating-default-category-issue/

### CODE REVIEW

**Comment by theoduplooy** [comment]
Great, so what happens now, what is the next step?

**Comment by Traumflug** [comment]
Now it's in the review queue, waiting for an experienced developer to look at it and test whether it works.

If you want to do something, apply these commits and tell us how well they work.

File: controllers/admin/AdminImportController.php [hunk 1]
```diff
@@ -2709,8 +2710,8 @@ protected function productImportOne($info, $idDefaultLanguage, $idLang, $forceId
             $product->id_category = array_values(array_unique($product->id_category));
         }
 
-        // Will update default category if there is none set here. Home if no category at all.
-        if (!isset($product->id_category_default) || !$product->id_category_default) {
+        // Will update default category if forced or if there is none set here. Home if no category at all.
+        if ($forceCat || !isset($product->id_category_default) || !$product->id_category_default) {
```
Comment (getdatakick) [inline controllers/admin/AdminImportController.php]: Variable `$forceCat` is undefined here. Did you forget to pass it to `productImportOne` method?

Comment (Zengraph) [reply controllers/admin/AdminImportController.php]: https://github.com/thirtybees/thirtybees/pull/1060/commits/f760274967338fcd49fcc2be6e8bc8c1ee4d75f3

Comment (getdatakick) [reply controllers/admin/AdminImportController.php]: That commit has nothing to do with the variable being missing / undefined here :)

**Comment by getdatakick** [comment]
I have used your code and fixed the undefined variable $forceCat. Picked via commit  e9f543b115ffaa1a5e52e82ba32a07a7f4938d90. Thanks for the contribution

**Pull request closed without merging**